### ISSUE TITLE
Fix license file

### ISSUE THREAD

**Issue by SarahAlidoost** [ESMValGroup/ESMValTool-website#10]
The license file [`LICENSE`](https://github.com/ESMValGroup/ESMValTool-website/blob/main/LICENSE) is related to the template. There is also a [`license.md` file](https://github.com/ESMValGroup/ESMValTool-website/blob/main/license.md) that is used for the content of the website. The file name should be changed, because `license.md` is recognized by GitHub as a license of this repository.

**Comment by axel-lauer** [comment]
Renaming of `license.md` is addressed here: https://github.com/ESMValGroup/ESMValTool-website/pull/28

**Comment by axel-lauer** [comment]
Is it fine to close this issue now?

**Comment by SarahAlidoost** [comment]
> Is it fine to close this issue now?

no, the license file [LICENSE](https://github.com/ESMValGroup/ESMValTool-website/blob/main/LICENSE) is `MIT` and related to the template and not to the repository https://github.com/ESMValGroup/ESMValTool-website. We should check if the license of this repo should be the same as the one for the template or if it can be `Apache 2.0`. I am not familiar with the `MIT` license.

**Comment by axel-lauer** [comment]
This website is based on the template "Beautiful Jekyll" (https://beautifuljekyll.com/). The way I understand the license of Beautiful Jekyll is that it requires their license and copyright notice to be included in any copy containing substantial parts of their code. From this, I might be wrong but I would think by using this template, we also accepted to have our stuff under the MIT license, which seems to be fine to me.

@ESMValGroup/userengagementteam any thoughts on this?
